index.html

```diff
@@ -0,0 +1,156 @@
+                    <li class="nav-list"><a href="#mainprogramme">Program</a></li>
+                    <li class="nav-list"><a href="#" class="join">Join</a></li>
+                    <li class="nav-list"><a href="#" class="sponsor">Sponsor </a></li>
+                    <li class="nav-list"><a href="#" class="news">News</a></li>
+                </ul>
+            </nav>
+
+        </section>
+
+        <article class="sec2">
+            <div class="mp" id="mainprogramme">
+                <h2 class="mainprogramme">Main Programme</h2>
+            </div>
+            <div class="programmecontainer">
+                <div class="cont">
+                    <img src="images/lecturer.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">lectures</h3>
+                    <div class="cont-description">Come get a talk from the most
+                        professional perspectives in the globe</div>
+                </div>
+                <div class="cont">
+                    <img src="images/exhibition.jpg" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Exhibition</h3>
+                    <div class="cont-description">Appreciate various art
+                        excellence on display organized from across
+                        continents</div>
+                </div>
+                <div class="cont">
+                    <img src="images/forum.jpg" alt="lecturer icon" width="70px"
+                        height="auto">
+                    <h3 class="cont-text">Forum</h3>
+                    <div class="cont-description">Come have the time to share
+                        your thoughts or questions with experts on
+                        whose answers you can rely </div>
+                </div>
+                <div class="cont">
+                    <img src="images/workshop.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Workshop</h3>
+                    <div class="cont-description">There will be a platformwhere
+                        you can build your own work whilst drawing
+                        inspiration from the live scenery</div>
+                </div>
+                <div class="cont">
+                    <img src="images/extra-badge-xxl.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Extras</h3>
+                    <div class="cont-description">There will be prizes to go
+                        around after the workshop and 3 lucky mentees
+                        taken under each of our guests's wings</div>
+                </div>
+            </div>
+
+            <nav class="wholepage">
+                <a href="" id="wholepage">SEE THE WHOLE PAGE</a>
+            </nav>
+
+        </article>
```

Comment:
Kindly add some spacing around this section as shown in the design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/191009444-459b32d0-a7d6-4c70-9b2a-f69a48868ff4.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/191009554-f13dc6fd-5f9c-4b13-b8c7-2dde350fd16d.png)